src/modules/dashboard/components/ColonyEvents/ColonyEventsListItem.css

```diff
@@ -108,3 +108,8 @@
 .titleDecoration {
   color: var(--pink);
 }
+
+.tooltip span {
+  vertical-align: text-top;
+  font-size: var(--size-tiny);
+}
```

Comment:
Just to keep it in line with the other tooltip styles (meaning they being bigger) please add:
```
  padding: 8px 4px;
```

To the styles

Reply:
If you are really feeling adventurous :) you can add this as a "theme" for the tooltip